shaders/palette.frag

```diff
@@ -16,24 +16,34 @@ uniform sampler2D remaps;
 
 in vec4 gl_FragCoord;
 
-bool REMAP_SPRITE = (options & 1u) != 0u;
+bool SPRITE_REMAP = (options & 1u) != 0u;
 bool SPRITE_SHADOWMASK = (options & 2u) != 0u;
 bool SPRITE_INDEX_ADD = (options & 4u) != 0u;
-bool USE_HAR_QUIRKS = (options & 8u) != 0u;
+bool SPRITE_HAR_QUIRKS = (options & 8u) != 0u;
+bool SPRITE_DARK_TINT = (options & 0x10u) != 0u;
```

Comment:
Could you add some comments on which things these are used for ? Would probably make it easier in the future to reason around the shader code.

Reply:
I'll add a comment directing the reader to video\enums.h (where those comments already live)